src/main/java/com/powsybl/openloadflow/network/LfBranch.java

```diff
@@ -139,12 +139,14 @@ default List<LfLimit> getLimits2(LimitType type) {
 
     double computeApparentPower2();
 
-    boolean isZeroImpedanceBranch(boolean dc, double lowImpedanceThreshold);
+    boolean isZeroImpedanceBranch();
 
     void setSpanningTreeEdge(boolean spanningTreeEdge);
 
     boolean isSpanningTreeEdge();
 
+    boolean isZeroImpedanceBranchWithEquation();
```

Comment:
I would remove this method and keep composite check isZeroImpedanceBranch() && isSpanningTreeEdge()